deployment/upgradeLive

```diff
@@ -6,31 +6,38 @@ set -e
 cd /opt/kaltura/liveController
 echo updating liveController v$1
 if [ ! -d "$1" ] ; then
-	if [ -L "${latest}" ]; then
-		echo "unlink ${latest}"
-		unlink ${latest}
-	fi
     echo Try to download  https://github.com/kaltura/liveDVR/archive/v$1.tar.gz
     wget https://github.com/kaltura/liveDVR/archive/v$1.tar.gz
     echo try to unzip v$1.tar.gz
     tar -xvzf v$1.tar.gz
     mv liveDVR-$1 $1
+    rm -rf /opt/kaltura/liveController/v$1.tar.gz
     cd $1
+    echo "current dir: [`pwd`]"
     nvm install
     # move configuration files (liveController's and liveRecorder's) to latest version
     cp /opt/kaltura/liveController/configMapping.json /opt/kaltura/liveController/$1/common/config/
     cp /opt/kaltura/liveController/configMapping.ini /opt/kaltura/liveController/$1/liveRecorder/Config/
     npm install
     cd ..
     # download binaries from Jenkins
+    echo "### starting to download binaries"
     bash /opt/kaltura/liveController/$1/deployment/get_bins "$1"
-    rm -rf /opt/kaltura/liveController/v$1.tar.gz
+    echo "### finished downloading binaries"
     # create symlinks to latest dir and process script
+	if [ -L "latest" ]; then
+		echo "unlink latest"
+		unlink latest
+	fi
     ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
-    ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+	if [ ! -L /etc/init.d/kLiveController ]; then
+		echo "ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController"
```

Comment:
I would change this to: "Creating symlink to kLiveController script at - /etc/init.d/kLiveController"
(or somthing like that)